IdentityCore/src/controllers/MSIDSilentController.m

```diff
@@ -96,7 +96,7 @@ - (void)acquireToken:(nonnull MSIDRequestCompletionBlock)completionBlock
     
     __auto_type request = [self.tokenRequestProvider silentTokenRequestWithParameters:self.requestParameters
                                                                          forceRefresh:self.forceRefresh];
-    
+    request.skipLocalRt = self.fallbackController != nil;
```

Comment:
Can this be made more explicit? I don't think having a fallback controller always means that we want to skip local RT

Reply:
Updated with a separate bool, and will only be assigned during creating silent Sso Ext Controller